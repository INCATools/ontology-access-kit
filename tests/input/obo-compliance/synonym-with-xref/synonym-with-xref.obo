format-version: 1.4
ontology: synonym-with-xref

[Term]
id: X:1
name: example term
synonym: "synonym with xref" EXACT [PMID:1]

