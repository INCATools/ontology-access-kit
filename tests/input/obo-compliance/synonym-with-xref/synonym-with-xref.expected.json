{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/synonym-with-xref.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "lbl" : "example term",
      "type" : "CLASS",
      "meta" : {
        "synonyms" : [ {
          "pred" : "hasExactSynonym",
          "val" : "synonym with xref",
          "xrefs" : [ "PMID:1" ]
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasDbXref",
      "lbl" : "database_cross_reference",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasExactSynonym",
      "lbl" : "has_exact_synonym",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ]
  } ]
}